Fugro/Test/Fugro.G2O.Test.csproj

```diff
@@ -1,18 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-
+    <TargetFrameworks>net6.0</TargetFrameworks>

```

Comment:
I tried... Since G2O is not multitarget (3 builds), running this multitarget fails (you need to run it on the solution, picking a configuration, but that than has only 1 framework built in it's output).

Tried in configuration-specific property groups to pick 1 target framework and then running 3 test calls in the github action for different configs, but it seems you can set targetframework in a config-specific property group. 

Anyway, tried, spent hours, gave up...

Reply:
Maybe it's possible with an exotic mix of not specifying target framework in the csproj, but in the test cli calls, etc, but then it won't work in VS IDE anymore. --> another hour went by, it will be another day to get this to work.

Maybe later?

Reply:
Sounds good to me.